cmd/kubeadm/app/cmd/init.go

```diff
@@ -323,6 +323,14 @@ func (i *Init) Run(out io.Writer) error {
 
 	adminKubeConfigPath := filepath.Join(kubeConfigDir, kubeadmconstants.AdminKubeConfigFileName)
 
+	// ReadSentineFile check sentinefile exist
+	if err := kubeadmutil.ReadSentineFile(); err != nil {
+		// WriteSentineFile is record kubeadmin init to sentinefile
+		if err := kubeadmutil.WriteSentineFile("kubeadm init"); err != nil {
+			glog.Errorf("[init] kubeadmin init record sentinefile error: %s\n", err)
```

Comment:
the message should look like:
`[init] cannot write sentinel file: %v, err`

the `err` should be formatted using `fmt.Errorf()` from sentinel.go.
and should include the sentinel path.

same for join.go.
[1]